Update create-eks-cluster-and-deploy-app.md (#1028)
port-labs/port-docs: docs/create-self-service-experiences/setup-backend/github-workflow/examples/AWS/create-eks-cluster-and-deploy-app.md

:::tip Starter Repository

> Clone our repository [here](https://github.com/port-labs/eks-deploy-guide) to follow along through the guide. The repository contains the following folders:

Reviewer: ```suggestion
Clone the repository [here](https://github.com/port-labs/eks-deploy-guide) to follow along with the guide. The repository contains the following folders:
```
